### PR TITLE
Add custom composer arguments support

### PR DESCRIPTION
**Pull request by sl0wik**
Because of `--no-dev` flag composer args are suitable only for production build. To allow dev or staging work `COMPOSER_ARGS` environment variable is introduced.

`COMPOSER_ARGS` can be used to override default composer args.

### CODE REVIEW

**Comment by sl0wik** [comment]
Perhaps I can add some more information on why this PR is important. AppEngine is a great tool in production but running apps locally was always complex. If you get a configuration mismatch between your local and production (for example with allowed functions, or dynamic modules), your app might work locally and pass unit tests, and then throw nasty 500 on production after the successful deployment.

For a long time, the only package that was providing environment comparable to App Engine production was https://github.com/GoogleCloudPlatform/php-docker . It was constantly outdated and poorly documented.

Buildpacks opened a new way to set up local and CI/CD environments equal to production, however `--no-dev` flag explicitly blocks the installation of necessary tools.

This simple PR (which might need quick testing), would essentially allow using buildpacks for all non-production environments and provide solution to most of issues in https://github.com/GoogleCloudPlatform/php-docker/issues .

File: pkg/php/php.go [hunk 1]
```diff
@@ -41,6 +42,8 @@ const (
 	dependencyHashKey = "dependency_hash"
 
 	composerVersionKey = "php"
+	// composerArgsEnv is an environment variable used to pass custom composer variables.
+	composerArgsEnv = "COMPOSER_ARGS"
```
Comment (pmaloogoogle) [inline pkg/php/php.go:46]: Could you please update the name to "GOOGLE_COMPOSER_ARGS" to follow the env var convention.

Comment (pmaloogoogle) [inline pkg/php/php.go:46]: Let's also move the new variable to gcp_buildpacks/pkg/env/env.go where we store all the env vars.

**Comment by sl0wik** [comment]
`COMPOSER_ARGS` removed to `GOOGLE_COMPOSER_ARGS`

File: pkg/php/php.go (hunk 1 above)
Comment (pmaloogoogle) [reply pkg/php/php.go]: thanks.

**Comment by pmaloogoogle** [comment]
I got formatting error in the php.go file blocking the merge. I believe they are:

1. Imports aren't sorted. "strings" import needs to be after the "path/filepath" import
2. there are extra trailing whitespaces on the line: `flags = []string{"--no-dev", "--no-progress", "--no-interaction", "--optimize-autoloader"}        `

you can also try running go format to see if there are other issues.

**Pull request merged**